components/style/matching.rs

```diff
@@ -401,14 +405,20 @@ trait PrivateMatchMethods: TElement {
         // in addition to the unvisited styles.
 
         let mut tasks = UpdateAnimationsTasks::empty();
-        if self.needs_animations_update(context, old_values.as_ref().map(|s| &**s), new_values) {
+        if self.needs_animations_update(
+            context,
+            old_values.as_ref().map(|s| &**s),
+            new_values,
+            None, /* pseudo_element */
```

Comment:
I think the most common syntax is `/* pseudo_element = */ None,` but again no big deal.

Reply:
:+1: